src/bindings/python/src/pyopenvino/frontend/manager.cpp

```diff
@@ -99,68 +99,68 @@ void regclass_frontend_FrontEndManager(py::module m) {
     });
 }
 
-OPENVINO_SUPPRESS_DEPRECATED_START
-
 // CVS-139194: migrate to new exception handling https://github.com/pybind/pybind11/pull/4772
 
 void regclass_frontend_GeneralFailureFrontEnd(py::module m) {
-    static py::exception<ov::frontend::GeneralFailure> exc(std::move(m), "GeneralFailure");
+    static py::handle ex = py::exception<ov::frontend::GeneralFailure>(std::move(m), "GeneralFailure").release();
     py::register_exception_translator([](std::exception_ptr p) {
         try {
             if (p)
                 std::rethrow_exception(p);
         } catch (const ov::frontend::GeneralFailure& e) {
-            exc(e.what());
+            py::set_error(ex, e.what());
         }
     });
 }
 
 void regclass_frontend_OpValidationFailureFrontEnd(py::module m) {
-    static py::exception<ov::frontend::OpValidationFailure> exc(std::move(m), "OpValidationFailure");
+    static py::handle ex =
+        py::exception<ov::frontend::OpValidationFailure>(std::move(m), "OpValidationFailure").release();
     py::register_exception_translator([](std::exception_ptr p) {
         try {
             if (p)
                 std::rethrow_exception(p);
         } catch (const ov::frontend::OpValidationFailure& e) {
-            exc(e.what());
+            py::set_error(ex, e.what());
```

Comment:
it's not available in older versions of pybind https://openvino-ci.toolbox.iotg.sclab.intel.com/blue/organizations/jenkins/private-ci%2Fie%2Fbuild-linux-macos_arm64/detail/build-linux-macos_arm64/15687/pipeline#step-402-log-99

as it's stated in tickets, we need to keep compatibility.

Reply:
done